src/main/java/frc/robot/OI.java

```diff
@@ -19,23 +19,21 @@ public class OI {
   private static final int operatorJoystickPort = 1;
 
   public static final class Driver {
-    private static final XboxController controller = new XboxController(driverJoystickPort);
+    public static final XboxController controller = new XboxController(driverJoystickPort);
```

Comment:
We decided that it was appropriate to shortcut this to public but that it was bad practice to work around the controller by adding public to it. When are we planning on addressing this?

Reply:
I do not know. But is not major enough of a issue to stop this PR